bin/watch_doc

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Added a small task to start up the doc server and auth update the docs when changes are made to the files within doc folder.